iam_policies.tf

```diff
@@ -359,6 +359,7 @@ data "aws_iam_policy_document" "shepherd_engineers" {
       "ssm:*",
       "tag:*",
       "vpc:*",
+      "ec2:*",
```

Comment:
Generally I try to keep these in alphabetical order.

Reply:
Reordered the policies, also added `rds:*` from what I assume is a dependency for RedShift. This issue came up in testing.